### DISCUSSION TITLE
Tooltip on icon hover

### DISCUSSION THREAD

**Discussion by v-c0de** [massCodeIO/massCode#80] [Ideas]
Possibility of adding a toggle to show/hide tooltips on the icons? 

![image](https://user-images.githubusercontent.com/65813722/168586631-9eacc59b-8807-48fe-a645-92b5687a27b7.png)

**Comment by antonreshetov** [comment]
@v-c0de in [v2.10.0](https://github.com/massCodeIO/massCode/releases/tag/v2.10.0) it's done.

**Reply by v-c0de** [reply]
thanks!
